src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -862,6 +862,10 @@ export class WorkspaceEdit implements vscode.WorkspaceEdit {
 					edit = editOrTuple;
 				}
 				if (NotebookEdit.isNotebookCellEdit(edit)) {
+					if (uri.scheme === CellUri.scheme) {
```

Comment:
@roblourens I don't think this is the best place to do the check. This should either be checked when creating a notebook editor or (better) when actually applying (but not while in "transport" as it is done here). 


I think this will be the better place for the scheme-check: https://github.com/microsoft/vscode/blob/e4d92055cfb9d87ac5620def57f1721e101ebb5f/src/vs/workbench/contrib/bulkEdit/browser/bulkCellEdits.ts#L67